Automattic/jetpack
# Integrate feedback export functionality with the feedback view filters

Fixes #27426.

#### Changes proposed in this Pull Request:
This PR integrates the export/download feedback functionality with the WP admin table by the following:

- A new second 'post' dropdown has been added at the top of the table in the filter section, which allows filtering feedback by the post from which it originates, in the same way as you'd select a post to download feedback from.
- An export button has been added at the top of the table, which by default will export all results matching the currently active filter. If any rows are selected, only those rows will be exported.
- The download feedback form at the bottom of the page has been removed as it's now obsolete.

Old export form:
![Screenshot 2022-11-15 at 22 13 20](https://user-images.githubusercontent.com/8056203/202028138-413b3966-5b09-4f9e-8c08-be425238d8ac.png)

New, integrated export form:
![Screenshot 2022-11-15 at 22 12 54](https://user-images.githubusercontent.com/8056203/202028149-0aea310a-f5cb-40c6-8d15-8bb793e9385c.png)

#### Other information:


#### Jetpack product discussion



pcsBup-oK-p2

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






- Go to WP Admin -> Feedback.
- You should now be able to filter your feedback responses by the post by choosing from the dropdown menu at the top of the table.
- Clicking 'Export' at the top of the table should export all responses matching the currently applied filter.
- Select some rows.
- When a selection is active, only the selected rows should be exported.
